int16[] position
string type
---
xyz real_position